actions/new.action.ts

```diff
@@ -115,12 +117,12 @@ export class NewAction extends AbstractAction {
       projectDirectory,
       shouldInstallMonitoring as boolean,
     );
-
-    await createMonitor(
+    
+    await createskipDocker(
```

Comment:
This approach is workable, but this feature should ideally be handled in combination with changes in the schematics as well since the bootstrapping is just one part of the system.

What happens when a user wants to generate monitoring related files? they would need to run two commands, `nest g monitor` and `nest g monitorModule` whereas it should ideally be implemented something as `nest g monitor --skipDocker`. The flag should be implemented at the level of each schematic, and while bootstrapping a new project, the `--skipDocker` flag should be propagated to each command